generator_test.go

```diff
@@ -45,6 +45,7 @@ func testNewV1(t *testing.T) {
 	t.Run("FaultyRand", testNewV1FaultyRand)
 	t.Run("MissingNetwork", testNewV1MissingNetwork)
 	t.Run("MissingNetworkFaultyRand", testNewV1MissingNetworkFaultyRand)
+	t.Run("V1Ordering", testNewV1Ordering)
```

Comment:
Please change `"V1Ordering"` to `"Ordering"`, since this is a subtest of `testNewV1`.